src/data/hooks.js

```diff
@@ -41,3 +45,16 @@ export function useTimeout(callback, delay) {
     timeoutIdRef.current = null;
   }, [callback, delay]);
 }
+
+export const useSelectedCourse = () => {
```

Comment:
Similarly, let's keep our hooks related to LCM close to the `learner-credit-management` module (e.g., `/src/components/learner-credit-management/data/hooks/useSelectedCourse`).

Reply:
Bump. The code for `useSelectedCourse` is still in `src/data/hooks.js` even though it was also added under the `learner-credit-management` module. Also, as mentioned in above comment, the convention under `/src/components/learner-credit-management/data/hooks/` is currently to have an individual file per React hook rather than having a `hooks.js` file (makes testing simpler). The intent is for the `/src/components/learner-credit-management/data/hooks/index.js` file to export all the independent hook files.

On second look, it also doesn't appear `useSelectedCourse` is used anywhere? Why was it introduced?